packages/application-components/src/components/modal-pages/custom-form-modal-page/README.md

```diff
@@ -0,0 +1,76 @@
+# CustomFormModalPage
+
+## Usage
+
+```js
+import CustomFormModalPageExample from './custom-form-modal-page.example';
+```
+
+## Description
+
+Custom Form Modal pages are controlled components used to render a page with a form or something that requires user input, similar to `InfoModalPage` but semantically with a different role. The header includes an area for control buttons, which can be customized at will. The component exposes three default buttons which can be useful for most use cases.
```

Comment:
Maybe we can be more clear here when to use which component?

Given that you intend to build yada
Then we recommend using an InfoModalPage
Given you intend to buildfoo
Then we recommend using an CustomFormModalPage

Not sure. Just a thought to have clearer usage guidelines.